.github/workflows/release-mainnet.yml

```diff
@@ -0,0 +1,18 @@
+name: Deploy Components to Mainnet
+
+on:
+  push:
+    branches: [main]
+
+jobs:
+  deploy-mainnet:
+    uses: nearbuilders/bos-workspace/.github/workflows/deploy.yml@main
+    with:
+      deploy-env: "mainnet"
+      bw-legacy: false
+      app-name: "BosGenaDrop"
+      deploy-account-address: ${{ vars.BOS_SIGNER_ACCOUNT_ID }}
+      signer-account-address: ${{ vars.BOS_SIGNER_ACCOUNT_ID }}
+      signer-public-key: ${{ vars.BOS_SIGNER_PUBLIC_KEY }}
```

Comment:
You don't need these to be stored on github, but if you already have it then no problem